dpnp/dpnp_iface_statistics.py

```diff
@@ -503,6 +570,20 @@ def correlate(a, v, mode="valid"):
         is ``"valid"``, unlike :obj:`dpnp.convolve`, which uses ``"full"``.
 
         Default: ``"valid"``.
+    method : {"auto", "direct", "fft"}, optional
+        `"direct"`: The correlation is determined directly from sums.
+
+        `"fft"`: The Fourier Transform is used to perform the calculations.
+        This method is faster for long sequences but can have accuracy issues.
+
+        `"auto"`: Automatically chooses direct or Fourier method based on
+        an estimate of which is faster.
```

Comment:
It would be better to format the description like:
```suggestion
        Specifies which method to use to calculate the correlation:

        - `"direct"` : The correlation is determined directly from sums.
        - `"fft"` : The Fourier Transform is used to perform the calculations.
          This method is faster for long sequences but can have accuracy issues.
        - `"auto"` : Automatically chooses direct or Fourier method based on
          an estimate of which is faster.
```
You can look into the docstring of `put_along_axis` as a reference.